src/DeserializationPojo/ListResourcePojo.java

```diff
@@ -1,20 +1,15 @@
-package pojo;
+package DeserializationPojo;
 
 import java.util.List;
 
-
-
-public class listUsers  {
-	
+public class ListResourcePojo {
```

Comment:
This represents an object that is returned only by list resource api. So we can name it `ListResourceResponsePojo`. The word Response makes it much more obvious what the class is.